src/main/catalog-sources/__test__/kubeconfig-sync.test.ts

```diff
@@ -151,7 +164,7 @@ describe("kubeconfig-sync.source tests", () => {
 
       fs.writeFileSync(filePath, contents);
```

Comment:
Opportunity to get rid of another `mock-fs` ;) Fix if feeling boy-scoutish :)

While we are at it, maybe these technical unit tests could be more awesome as part of behavioural tests instead. WDYT? :)

Reply:
Discussion handled IRL.